plugins/out_stackdriver/stackdriver.c

```diff
@@ -367,7 +367,8 @@ static struct mk_list *parse_local_resource_id_to_list(char *str, char *type)
  *    that are specified in the configuration of stackdriver_out plugin
  */
 static int process_local_resource_id(const void *data, size_t bytes,
-                                     struct flb_stackdriver *ctx, char *type)
+                                     struct flb_stackdriver *ctx, char *type,
```

Comment:
Instead of passing tag to this function, could you split the logic into two functions:
1. init local_resource_id: read local_resource_id from *data or *tag
2. process local_resource_id: like what you have done in this function for extracting the related pod/container/node names.

Reply:
updated